client/src/components/Auction.jsx

```diff
@@ -63,8 +63,9 @@ class Auction extends Component {
           } else {
             answer.json()
             .then((bid) => {
-              console.log(bid);
+              console.log(bid.current_bid);
```

Comment:
is this console.log still useful for your future work? if not you should consider deleting it.